.github/actions/build_deb.sh

```diff
@@ -2,33 +2,67 @@
 
 echo "Building the SIL Kit packages"
 
-orig_tarball=${SILKIT_SOURCE_URL}/${SILKIT_VERSION}-${SILKIT_VERSION_SUFFIX}/libsilkit_${SILKIT_VERSION}.orig.tar.xz
-wget --no-proxy --no-check-certificate $orig_tarball
-ret_val=$?
-if [ "$ret_val" != '0' ] ; then
-    echo "SILKIT-PKG: Could not get orig tarball from $orig_tarball"
-    exit 64
+if [ -z $SILKIT_PKG_URL ] ; then
+    SILKIT_PKG_URL="."
 fi
+debian_path="${SILKIT_PKG_URL}/debian"
```

Comment:
Not planned, the reason was due to maintenance reasons. dh_make+debbuild really, really want to build the package from a tarball that is separate from the `debian` directory. So it can get complicated with version management if we add patches or changes to the debian directory.